ajax/libs/printThis/package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "printThis",
+  "description": "Printing plug-in for jQuery",
+  "filename": "printThis.min.js",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/jasonday/printThis.git"
+  },
+  "keywords": [
+    "print",
+    "jQuery",
+    "page title",
```

Comment:
We prefer not to separated words in `keywords` exception the condition it needs more than one word to represent the meaning. Just suggestion for you: DOM, styling, plugin can be added. You can confirm if they are good keywords by googling them, the sequence you input also has an impact on the search result. For example, we will see different result after inputing `Print DOM styling jQuery plugin` and `Print jQuery DOM styling  plugin`